### PR TITLE
[SMALLFIX]Change the AllocatorFactory.java  using "static"

### PR DESCRIPTION
**Pull request by naive4E4A55**

### CODE REVIEW

File: core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java [hunk 1]
```diff
@@ -10,14 +10,16 @@
  */
 
 package alluxio.worker.block.allocator;
+import static org.junit.Assert.assertTrue;
```
Comment (apc999) [inline core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java]: space above

File: core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java [hunk 2]
```diff
@@ -10,14 +10,16 @@
...
 
 import alluxio.conf.ServerConfiguration;
 import alluxio.conf.PropertyKey;
 import alluxio.worker.block.BlockMetadataManagerView;
 import alluxio.worker.block.TieredBlockStoreTestUtils;
 
 import org.junit.After;
-import org.junit.Assert;
+//import org.junit.Assert;
```
Comment (apc999) [inline core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java]: remove line 21

File: core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java [hunk 3]
```diff
@@ -56,7 +58,8 @@ public void after() {
   public void createGreedyAllocator() {
     ServerConfiguration.set(PropertyKey.WORKER_ALLOCATOR_CLASS, GreedyAllocator.class.getName());
     Allocator allocator = Allocator.Factory.create(mManagerView);
-    Assert.assertTrue(allocator instanceof GreedyAllocator);
+    //Assert.assertTrue(allocator instanceof GreedyAllocator);
```
Comment (apc999) [inline core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java]: remove

File: core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java [hunk 4]
```diff
@@ -67,7 +70,8 @@ public void createGreedyAllocator() {
   public void createMaxFreeAllocator() {
     ServerConfiguration.set(PropertyKey.WORKER_ALLOCATOR_CLASS, MaxFreeAllocator.class.getName());
     Allocator allocator = Allocator.Factory.create(mManagerView);
-    Assert.assertTrue(allocator instanceof MaxFreeAllocator);
+    //Assert.assertTrue(allocator instanceof MaxFreeAllocator);
```
Comment (apc999) [inline core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java]: remove

File: core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java [hunk 5]
```diff
@@ -79,7 +83,8 @@ public void createRoundRobinAllocator() {
     ServerConfiguration.set(PropertyKey.WORKER_ALLOCATOR_CLASS,
         RoundRobinAllocator.class.getName());
     Allocator allocator = Allocator.Factory.create(mManagerView);
-    Assert.assertTrue(allocator instanceof RoundRobinAllocator);
+    //Assert.assertTrue(allocator instanceof RoundRobinAllocator);
```
Comment (apc999) [inline core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java]: remove

File: core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java [hunk 6]
```diff
@@ -91,6 +96,7 @@ public void createDefaultAllocator() {
     // Create a new instance of Alluxio configuration with original properties to test the default
     // behavior of create.
     Allocator allocator = Allocator.Factory.create(mManagerView);
-    Assert.assertTrue(allocator instanceof MaxFreeAllocator);
+    //Assert.assertTrue(allocator instanceof MaxFreeAllocator);
```
Comment (apc999) [inline core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorFactoryTest.java]: remove

**Comment by newnius** [comment]
@apc999 Duplicate https://github.com/Alluxio/alluxio/pull/9075

**Pull request closed without merging**